src/main/fc/parameter_names.h

```diff
@@ -168,6 +168,12 @@
 #define PARAM_NAME_ALTITUDE_I "autopilot_altitude_I"
 #define PARAM_NAME_ALTITUDE_D "autopilot_altitude_D"
 #define PARAM_NAME_ALTITUDE_F "autopilot_altitude_F"
+#define PARAM_NAME_POSITION_P "autopilot_position_P"
+#define PARAM_NAME_POSITION_I "autopilot_position_I"
+#define PARAM_NAME_POSITION_D "autopilot_position_D"
+#define PARAM_NAME_POSITION_A "autopilot_position_A"
+#define PARAM_NAME_POSITION_CUTOFF "autopilot_position_cutoff"
+#define PARAM_NAME_AP_MAX_ANGLE "autopilot_max_angle"
```

Comment:
```suggestion
#define PARAM_NAME_AP_ALTITUDE_I "autopilot_altitude_I"
#define PARAM_NAME_AP_ALTITUDE_D "autopilot_altitude_D"
#define PARAM_NAME_AP_ALTITUDE_F "autopilot_altitude_F"
#define PARAM_NAME_AP_POSITION_P "autopilot_position_P"
#define PARAM_NAME_AP_POSITION_I "autopilot_position_I"
#define PARAM_NAME_AP_POSITION_D "autopilot_position_D"
#define PARAM_NAME_AP_POSITION_A "autopilot_position_A"
#define PARAM_NAME_AP_POSITION_CUTOFF "autopilot_position_cutoff"
#define PARAM_NAME_AP_MAX_ANGLE "autopilot_max_angle"
```

Reply:
I plan on doing this renaming in the followup PR which tidies up the GPS rescue code to work with the new Autopilot functions.  Definitely agree with the suggestion above and will implement it, for sure.